src/utils.cc

```diff
@@ -37,7 +37,7 @@ Matcher::Matcher(const std::string &pattern)
                                         std::regex_constants::optimize);
 }
 
-Matcher::~Matcher() {}
+Matcher::~Matcher() = default;
```

Comment:
Not sure how `= default` helps.

Reply:
They are identical, using `= default` is more modern (diagnosed via `clang-tidy` with `modernize-use-default-member-init` option

Reply:
Can't we just let the compiler generate a destructor for us, and remove the declaration?